scripts/release.mjs

```diff
@@ -56,31 +59,68 @@ const UID = crypto.randomBytes(4).toString('hex');
 if (CURRENT_BRANCH !== DEFAULT_BRANCH) {
     BUMP = 'prerelease';
     DIST_TAG = 'alpha';
-    await $`npm version ${ BUMP } --preid=${ DIST_TAG }-${ UID }`;
+    if (DRY_RUN) {
+        await $`npm --no-git-tag-version version ${ BUMP } --preid=${ DIST_TAG }-${ UID }`;
+    } else {
+        await $`npm version ${ BUMP } --preid=${ DIST_TAG }-${ UID }`;
+    }
```

Comment:
Can we use a ternary to reduce one extra branching due to `if`


```suggestion
    await $`npm ${ dryRunGitOption } version ${ BUMP }`;
   
```

Reply:
@ravishekhar, thanks for the feedback! If we use variables instead of branching, we'll need to use two different variables, one for `npm --no-git-tag-version version` and another for `npm publish --dry-run`. We'll also still need to branch on lines 75 & 100. Which approach do you prefer?

Reply:
I am inclining towards having more variables than having more conditional statements. Do you have any other thoughts here?

Reply:
@ravishekhar, yeah, this sounds good to me. I just pushed the changes. They should be ready for a final review. 🤞